playbooks/tofino/setup_nodes-lab_trial.yml

```diff
@@ -31,6 +31,13 @@
   vars:
     topo_dict: "{{ lookup('file','{{ topo_file_loc }}') | from_yaml }}"
 
+# Startup ElasticSearch and Kibana
+#   note: if done after switch startup the wait value needs to be 180 for taking
+#   in configuration
+- import_playbook: ../ae/start_ae_elk.yml
```

Comment:
The setup_ae_udp_pipeline or setup_ae_tcp_pipeline needs to be called depending on scenario to be invoked to setup AE